test/integration/freetext_editor_spec.js

```diff
@@ -32,6 +32,13 @@ describe("Editor", () => {
       await closePages(pages);
     });
 
+    const countStorageEntries = async page =>
+      page.evaluate(
+        () =>
+          window.PDFViewerApplication.pdfDocument.annotationStorage._storage
+            .size
```

Comment:
There's already an official getter for this, so please don't access a "private" property like this (since that'll break if we convert `AnnotationStorage` to actual *private* properties).
```suggestion
          window.PDFViewerApplication.pdfDocument.annotationStorage.size
```